### PR TITLE
Charts: add skeleton for a new JS package

### PR DESCRIPTION
**Pull request by jeherve**
## Proposed changes:

This package can then be used by multiple teams to handle charts within Automattic properties.

- [x] I created the matching mirror repo already.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

* pfKYZu-1Jg-p2#comment-1530

## Does this pull request change what data or activity we track or use?
* No

## Testing instructions:

Not much to test yet. Is CI happy?

### CODE REVIEW

**Review by anomiex — DISMISSED** [review]
Looks ok as a skeleton

**Comment by jeherve** [comment]
@Automattic/jetpack-garage I had to push a few additional changes after running the `generate` command. I think it may be worth updating `generate` so those are created right off the bat?

File: projects/js-packages/charts/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+	"name": "@automattic/jetpack-charts",
```
Comment (oskosk) [inline projects/js-packages/charts/package.json]: I guess it should be @automattic/charts here too

Comment (kraftbj) [reply projects/js-packages/charts/package.json]:
```suggestion
	"name": "@automattic/charts",
```

**Review by kraftbj — APPROVED** [review]

**Pull request merged**